tests/conftest.py

```diff
@@ -340,6 +344,24 @@ def app_config(database):
     return cfg
 
 
+@pytest.fixture(scope="session")
+def app_config(database):
+
+    return get_app_config(
+        database,
+    )
```

Comment:
style: Odd that the linter didn't try to inline this to a single line 🤔

Reply:
My eyes didn't catch this, neither... I guess I _blindly_ rely on auto-linting :innocent: